xinference/core/chat_interface.py

```diff
@@ -12,31 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import base64
 import logging
 import os
+from io import BytesIO
 from typing import Generator, List, Optional
 
 import gradio as gr
+import PIL.Image
```

Comment:
Add this package to required dependencies

Reply:
I added pillow to the install requires. Thanks.